.gitignore

```diff
@@ -92,7 +92,7 @@ ipython_config.py
 #Pipfile.lock
 
 # PEP 582; used by e.g. github.com/David-OConnor/pyflow
-__pypackages__/
+_pypackages_/
```

Comment:
Is it really needed

Reply:
![image](https://user-images.githubusercontent.com/93659937/168774820-5a17945b-c48e-4cb4-b71e-1b20dde4c4fd.png)
we have this line in the main code
@ronyyosef

Reply:
ok, why are you changing that line?

Reply:
Not on purpose
This may have happened while installing a directory
I returned to the original